generators/angular/templates/src/main/webapp/app/entities/_entityFolder/_entityFile.model.ts.ejs

```diff
@@ -18,11 +18,18 @@
 -%>
 <%
 const enumImports = this.generateEntityClientEnumImports(fields);
+const predicate = (relationship) => {
```

Comment:
@RamiSouai Maybe we should give the predicate better name to clarify what it filters.